nano/node/testing.cpp

```diff
@@ -234,7 +235,8 @@ void nano::system::generate_activity (nano::node & node_a, std::vector<nano::acc
 
 nano::account nano::system::get_random_account (std::vector<nano::account> & accounts_a)
 {
-	auto index (random_pool.GenerateWord32 (0, accounts_a.size () - 1));
+	assert (std::numeric_limits<CryptoPP::word32>::max () > accounts_a.size ());
```

Comment:
release_assert seems to be used other places for std::numeric_limits<CryptoPP::word32>::max () checks. Maybe it should be `assert` everywhere?

Reply:
`assert` was used here because it is only used by the tests. I think `release_assert` makes sense in the other places as bad things will happen in production if it overflows regardless.